docs/api/plugins.rst

```diff
@@ -54,31 +37,15 @@ hello.py
 
     BaseSheet.addCommand('0', 'hello-world', 'status(options.disp_hello)')
 
-plugins.jsonl
-^^^^^^^^^^^^^^^^
-
-::
+If planning to open a PR on the VisiData repo, make sure the plugin contains:
 
-    {
-     "name": "helloworld",
-     "description": "a configurable greeting on demand",
-     "maintainer": "Vi Sidata <vd@example.com>",
-     "latest_release": "2020-06-11",
-     "url": "https://raw.githubusercontent.com/saulpw/visidata/387de72b369039ae864f1b84d8191d1085c2105b/plugins/hello.py",
-     "latest_ver": "1.0",
-     "visidata_ver": "2.0",
-     "pydeps": "",
-     "vdplugindeps": "",
-     "sha256": "6b3baf4e1c4550947e611b9f1bdc96b7193f85048d0fc83880c4be0e7a5537d4"
-    }
-
-- Use ``sha256 hello.py`` to compute the hash.
+* a ``__version__`` of the current release
```

Comment:
We're not versioning plugins in the mono-repo any more.  They can add a `__author__` and `__description__` however.